### ISSUE TITLE
AmbiguousMatch exception using Tensor GetArrayString method

### ISSUE THREAD

**Issue by eerhardt** [Microsoft/workbooks#405]
### Steps to Reproduce

0. Add these NuGet feeds 
    1. `https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json`
    2. `https://dotnet.myget.org/F/dotnet-core/api/v3/index.json`
1. Open this workbook and run it

````
---
uti: com.xamarin.workbook
id: 6b012591-182a-4235-9326-c8082ef94024
title: Tensors
platforms:
- DotNetCore
packages:
- id: System.Numerics.Tensors
  version: 0.1.0-preview2-180607-3
---

```csharp
#r "System.Numerics.Tensors"
```

```csharp
using System.Numerics;

Tensor<long> foo = new CompressedSparseTensor<long>(
                new long[] { 1, 2, 3, 4 }, 
                new int[] { 0, 3, 4, 4 }, 
                new int[] { 0, 1, 2, 2 }, 
                4, 
                new int[] { 8, 3 });

Console.WriteLine(foo.GetArrayString());
```
````

### Expected Behavior
It should print the string correctly.

**Update**: It appears my code is throwing an exception.  So in this case I expect to get the exception my code threw printed correctly, which will help me fix my bug.

### Actual Behavior
```

AmbiguousMatchException: Ambiguous match found.
at PropertyInfo System.RuntimeType.GetPropertyImpl (string name, BindingFlags bindingAttr, Binder binder, Type returnType, Type[] types, ParameterModifier[] modifiers)
at PropertyInfo System.Type.GetProperty (string name, BindingFlags bindingAttr)
at PropertyInfo Xamarin.Interactive.Representations.Reflection.StackFrame.GetPropertyForMethodAccessor (MethodBase method)
at StackFrame Xamarin.Interactive.Representations.Reflection.StackFrame.Create (StackFrame frame)
at TResult[] System.Linq.Enumerable.SelectArrayIterator<TSource, TResult>.ToArray ()
at TSource[] System.Linq.Enumerable.ToArray<TSource> (IEnumerable<TSource> source)
at StackTrace Xamarin.Interactive.Representations.Reflection.StackTrace.Create (StackTrace trace)
at ExceptionNode Xamarin.Interactive.Representations.Reflection.ExceptionNode.Create (Exception exception)
at void Xamarin.Interactive.CodeAnalysis.EvaluationContext.<RunAsync>d__26.MoveNext ()
at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw ()
at void System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (Task task)
at void Xamarin.Interactive.CodeAnalysis.Compilation.<HandleAsync>d__26.MoveNext ()
at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw ()
at void System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (Task task)
at void Xamarin.Interactive.Core.MainThreadRequest<TResponseMessage>.<>c__DisplayClass6_0.<<Handle>b__0>d.MoveNext ()
```
#### Xamarin Workbooks
| Detail | Value |
| :- | :- |
| Version | 1.4.1-dev+57 |
| Git Branch | 1.4-release |
| Git Hash | 923a8c9 |
| VSTS Definition | Workbooks & Inspector (Win) |

#### System Info
| Component | Value |
| :- | :- |
| Windows | Microsoft Windows NT 10.0.17134.0 (64-bit) |
| CPU Cores | 8 |
| Physical Memory | 32 GB |

#### VisualStudio.15.IntPreview Components
| Component | Version |
| :- | :- |
| Visual Studio Enterprise 2017 | 15.8.0-pre.3.0+27804.1.d15.8 |

### Log File

[Xamarin Workbooks 2018-06-07.log](https://github.com/Microsoft/workbooks/files/2081277/Xamarin.Workbooks.2018-06-07.log)

**Comment by abock** [comment]
Looking into this, but I'm getting a package install error against .NET Core 2.1 (which our soon-to-release 1.5 supports):

> Unable to find a version of 'System.Memory' that is compatible with 'System.Numerics.Tensors 0.1.0-preview2-180706-2 constraint: System.Memory (>= 4.6.0-preview1-26614-04)'.

Aside: I've edited the summary to fix the workbook sample 😄

> Note: Not sure how to put triple backticks in a code block. You'll need to add the extra backticks above.

Just use a larger outer fence:

`````
````
```
```
````
`````

**Comment by eerhardt** [comment]
I've fixed the repro steps above. I forgot that you also need the `https://dotnet.myget.org/F/dotnet-core/api/v3/index.json` NuGet feed to install this package.

**Comment by abock** [comment]
@eerhardt The exception rendering will be fixed in 1.5.0 which should come out on Monday. It's handled by #485.

The real exception you want to see is:

```
IndexOutOfRangeException: Index was outside the bounds of the array.
at bool System.Numerics.CompressedSparseTensor<T>.TryFindIndex (int compressedIndex, int nonCompressedIndex, out int valueIndex)
at T System.Numerics.CompressedSparseTensor<T>.get_Item (ReadOnlySpan<int> indices)
at T System.Numerics.Tensor<T>.get_Item (int[] indices)
at string System.Numerics.Tensor<T>.GetArrayString (bool includeWhitespace)
```

**Comment by sandyarmstrong** [comment]
Fixed in 1.4.3.
